Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Blender/AddOns/SceneExporter/__init__.py

```diff
@@ -33,10 +33,13 @@
 # @section Launcher Notes
 # - Comments are Doxygen compatible
 
+# To DO: Check for invalid Bone.oo1 names. Fix the not selected bug. No we need Bone Ends?
```

Comment:
What does "No we need Bone Ends?" mean?

Reply:
Oh..I will take that out on my next PR. Yeah, we do not want Leaf bones to ends added.. they are not necessary and if anything annoying.